test-curves/src/bls12_381/mod.rs

```diff
@@ -22,12 +24,16 @@ pub mod g2_swu_iso;
 pub use {fq::*, fq12::*, fq2::*, fq6::*, g1::*, g1_swu_iso::*, g2::*, g2_swu_iso::*};
 
 #[cfg(test)]
+#[cfg(feature = "bls12_381_curve")]
```

Comment:
Since Fq is only loaded when `bls12_381_curve`, this effectively means that all the tests would require `bls12_381_curve`.